charts/argo-controller/templates/config.yaml

```diff
@@ -49,5 +49,13 @@ data:
       secondsAfterSuccess: {{ .workflowTTL.secondsAfterSuccess }}
       secondsAfterFailure: {{ .workflowTTL.secondsAfterFailure }}
     {{- end }}
+  config: |
```

Comment:
You don't need the `config:` prefix (they added support for dropping it in 2.7).